lib/index.js

```diff
@@ -89,7 +89,11 @@ internals.implementation = (server, options) => {
                 }
             }
 
-            const [tokenType, token] = authorization.split(/\s+/);
+            if(authorization.split(/\s+/).length === 2) const [tokenType, token] = authorization.split(/\s+/);
```

Comment:
@artentica the build is not reporting this because it hides test results run the following
```sh
npx lab -a code -t 100 -L -r lcov -r console --verbose
```
output:
```
20 of 39 tests failed
Test duration: 339 ms
Assertions count: 39 (verbosity: 1.00)
No global variable leaks detected
Coverage: 71.07% (35/121)
lib/index.js missing coverage on line(s): 69, 75, 85, 86, 87, 88, 92, 95, 96, 99, 100, 102, 105, 106, 108, 109, 112, 113, 116, 118, 120, 121, 122, 123, 126, 127, 129, 130, 134, 136, 137, 138, 141, 142, 145
Code coverage below threshold: 71.07 < 100
Linting results:
        /home/ubuntu/workspace/hapi-now-auth/lib/index.js:
                Line 92: keyword-spacing - Expected space(s) after "if".
                Line 93: no-unused-vars - 'tokenType' is assigned a value but never used. Allowed unused vars must match /^internals$/.
                Line 93: no-unused-vars - 'token' is assigned a value but never used. Allowed unused vars must match /^internals$/.
                Line 94: brace-style - Closing curly brace appears on the same line as the subsequent block.
                Line 95: no-undef - 'token' is not defined.
                Line 95: semi - Missing semicolon.
                Line 96: no-undef - 'tokenType' is not defined.
                Line 96: semi - Missing semicolon.
                Line 99: no-undef - 'token' is not defined.
                Line 100: no-undef - 'tokenType' is not defined.
                Line 112: no-undef - 'token' is not defined.
                Line 122: no-undef - 'token' is not defined.
                Line 134: no-undef - 'token' is not defined.
                Line 134: no-undef - 'token' is not defined.
```
this line is not exposing the variables `tokenType` and `token` because they are scoped into the `if` statement but are required in downstream execution.

Reply:
I have make a modification, i'll try too run correctly npx too test (have a little bit a difficulty right now =) )and push you than again ^^